crates/weaver_resolver/src/registry.rs

```diff
@@ -793,23 +809,37 @@ mod tests {
             )
             .expect("Failed to deserialize expected attribute catalog");
 
+            // Write observed output.
+            let observed_attr_catalog_file = OpenOptions::new()
+                .create(true)
+                .write(true)
+                .open(observed_output_dir.join("attribute-catalog.json"))
+                .expect("Failed to open observed output file");
+            serde_json::to_writer_pretty(observed_attr_catalog_file, &observed_attr_catalog)
+                .expect("Failed to write observed ouptut.");
+            // Compare values
```

Comment:
What is the purpose of this? Debugging?

Reply:
no, updating the test values for some of these really large files is easier to look at the new and validate, then copy over.

Reply:
I see. Thanks